pylanguagetool/__init__.py

```diff
@@ -48,6 +49,13 @@ def init_config():
     p.add_argument("--enabled-only", action='store_true', default=False,
                    help="enable only the rules and categories whose IDs are specified with --enabled-rules or --enabled-categories"
                    )
+    p.add_argument(
+        '--pwl', '--private-word-list',
```

Comment:
Great idea, but I think `personal-word-list` would be a bit better name (I think it is also called that in the Languagetool browser extension).

Reply:
Very good point, renamed it